region/crateria/east.json

```diff
@@ -1329,13 +1382,41 @@
                   "notable": false,
                   "requires": [
                     "canSuitlessMaridia",
+                    "HiJump",
+                    "canWalljump"
+                  ]
+                },
+                {
+                  "name": "Suitless with Space Jump",
+                  "notable": false,
+                  "requires": [
+                    "canSuitlessMaridia",
+                    "SpaceJump",
                     {"or": [
                       "HiJump",
```

Comment:
With the proposed adjustments to the Base Suitless strat, hjb would become redundant here. So we could replace this `or` with just `canWalljump`.